src/test/java/org/prebid/server/bidder/dmx/DmxBidderTest.java

```diff
@@ -225,6 +263,27 @@ public void makeHttpRequestsShouldUpdateImpTagIdFromDmxIdWhenExtTagIdAndDmxIdIsP
                 .containsExactly("dmxId");
     }
 
+    @Test
+    public void makeHttpRequestsShouldModifyImpVideoWithNeededProtocolsIfNotSupplied() {
+        // given
+        final BidRequest bidRequest = givenBidRequest(
+                builder -> builder.video(Video.builder().protocols(null).build()));
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = dmxBidder.makeHttpRequests(bidRequest);
+
+        final List<Integer> expectedProtocols = new ArrayList<>(Arrays.asList(2, 3, 5, 6, 7, 8));
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).hasSize(1)
+                .extracting(HttpRequest::getPayload)
+                .flatExtracting(BidRequest::getImp)
+                .extracting(Imp::getVideo)
+                .flatExtracting(Video::getProtocols)
+                .containsAll(expectedProtocols);
```

Comment:
.containsAll(Arrays.asList(2, 3, 5, 6, 7, 8));

Reply:
I would like to keep variable `expectedProtocols` so this won't be looking like magic-values in assertion. Thoughts @snahornyi ?